packages/elements/src/components/ino-snackbar/ino-snackbar.tsx

```diff
@@ -56,10 +56,10 @@ export class Snackbar implements ComponentInterface {
   @Prop() timeout?: number = 5000;
 
   /**
-   * The role of the dialog. Can be either 'dialog' or 'alertdialog'.
+   * The role of the dialog. Can be either 'status', 'alert' or 'alertdialog'.
    * The 'alertdialog' role should be used for important messages that requires user interaction.
    */
-  @Prop() snackbarRole?: 'alert' | 'alertdialog' = 'alert';
+  @Prop() snackbarRole?: 'status' | 'alert' | 'alertdialog';
```

Comment:
Do we still need this property when we can just infer the correct role by ourselves? I would suggest removing it entirely.